internal/cmd/connect/command_cluster.go

```diff
@@ -48,6 +48,10 @@ func (c *clusterCommand) validArgs(cmd *cobra.Command, args []string) []string {
 		return nil
 	}
 
+	return c.validArgsMultiple(cmd, args)
```

Comment:
Doesn't the check on line 47 guarantee that `len(args) == 0`?

Reply:
Yeah. The only difference between the autocompletion code for commands which take one or multiple arguments is the check on line 47. If it takes one arg, then it only autocompletes if there are no args. Otherwise, it autocompletes regardless.

So I'm saving some lines by having one call the other. I did realize that the naming is unfortunate, though.

What about `validArg` and `validArgs`, with `validArg` having the length check and calling `validArgs`?

Reply:
Oh, I see. This took me a while to understand, but I think the current naming makes sense.